ark-cli/src/commands/collisions.rs

```diff
@@ -0,0 +1,20 @@
+use std::path::PathBuf;
+
+use crate::{monitor_index, AppError};
+
+#[derive(Clone, Debug, clap::Args)]
+#[clap(
+    name = "collisions",
+    about = "Find collisions in the ark managed folder"
+)]
+pub struct Collisions {
+    #[clap(value_parser, help = "Path to the root directory")]
+    root_dir: Option<PathBuf>,
+}
+
+impl Collisions {
+    pub fn run(&self) -> Result<(), AppError> {
+        // FIXME: How does `monitor_index` handle `ark-cli collisions`?
+        monitor_index(&self.root_dir, None)
+    }
+}
```

Comment:
We might not need `collisions` command now because we have `list`.

We could implement these features for `list` command:
1. Reporting collisions number.
2. Reconstructing lists of all collided paths.
3. Grouping the collided paths by ids.

Reply:
In the planned changes for #31, I suggest handling collisions in a separate method within `ResourceIndex`. This approach will prevent us from managing collisions during each insert, update, or remove operation.

We can then modify the collisions command in `ark-cli` to display the output of the new method in a well-formatted table.


What do you think?

Reply:
Let's remove the `FIXME` before merging. Feel free to create a follow-up issue if necessary.